templates/rhdm77-authoring-ha.yaml

```diff
@@ -860,7 +860,7 @@ objects:
             value: "${KIE_MBEANS}"
 ## OpenShift Enhancement BEGIN
           - name: KIE_SERVER_CONTROLLER_OPENSHIFT_ENABLED
-            value: "false"
+            value: "true"
```

Comment:
The file templates/docs/rhdm77-authoring-ha.adoc should be updated accordingly to state "KIE_SERVER_CONTROLLER_OPENSHIFT_ENABLED" parameter from "false" to "true".